src/test/resources/fixtures/hmh/hmh-ri/development/caliperEventGradeGraded.json

```diff
@@ -0,0 +1,65 @@
+{
+	"@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+	"id": "urn:uuid:a50ca17f-5971-47bb-8fca-4e6e6879001d",
+	"type": "GradeEvent",
+	"actor": {
+		"id": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+		"type": "Person"
+	},
+	"action": "Graded",
+	"object": {
+		"id": "urn:uuid:1050e852-5edb-4743-92d1-b53466de3a5f",
+		"type": "Attempt",
+		"assignee": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+		"assignable": {
+			"id": "urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+			"type": "Assessment"
+		},
+		"count": 35,
+		"dateCreated": "2016-11-15T10:05:00.000Z",
+		"startedAtTime": "2016-11-15T10:05:00.000Z",
+		"endedAtTime": "2016-11-15T10:55:12.000Z",
+		"duration": "PT1200S"
+	},
+	"eventTime": "2016-11-15T10:57:06.000Z",
+	"edApp": {
+		"id": "https://www.hmhco.com/ReadingInventory",
+		"type": "SoftwareApplication",
+		"extensions": {
+			"applicationCode": "RI",
+			"disciplineCode": "ED18_RLA"
+		}
+	},
+	"generated": {
+		"id": "urn:uuid:2050e852-5edb-4743-92d1-b53466de3a5f",
+		"type": "Score",
+		"attempt": "urn:uuid:1050e852-5edb-4743-92d1-b53466de3a5f",
+		"maxScore": 1700,
+		"scoreGiven": 800,
+		"scoredBy": "https://www.hmhco.com/ReadingInventory",
+		"comment": "The lexile score from Test Quiz # c35603f7a3434e2cb74ef14d60e70f42 taken on date/time for student 0f4fedbe-2227-415f-8553-40731a627171",
+		"dateCreated": "2016-11-15T10:56:00.000Z"
+	},
+    "membership": {
+        "id": "urn:uuid:randomGuidId",
+        "type": "Membership",
+        "organization": {
+            "id": "urn:uuid:sdfhs76h-2343-a087-34b3-df423kb2zjz8",
+            "type": "CourseSection",
+            "academicSession": "BOY 2019",
```

Comment:
Ed Analytics only supports the Organization fields and not the fields specific to subtypes, in this case CourseSection, the field 'academicSession' will be ignored and not picked up by the pipeline.